src/emqx_misc.erl

```diff
@@ -26,6 +26,7 @@
 -export([ merge_opts/2
         , maybe_apply/2
         , maybe_mute_rpc_log/0
+        , safe_io_device/0
```

Comment:
Q: shouldn't this be called in nodetool?

Reply:
no, it is to be called by the rpc handler node.
only one place found so far: emqx_license (will be sent in another PR to emqx-enterprise.git).